src/script.ts

```diff
@@ -20,6 +20,7 @@ addColorScheme(value)
 w['<%= options.globalName %>'] = {
   preference,
   value,
+  disableTransition: '<%= options.disableTransition %>',
```

Comment:
Why forcing it to be a string here?

Reply:
I have no idea how to use `boolean` type when `addTemplate` processes the file. 

Perhaps `'<%= options.disableTransition %>' === 'true'` is an optional solution?